code_schemes/s01e01.json

```diff
@@ -1,8 +1,112 @@
 {
   "SchemeID": "Scheme-03bb299c",
-  "Name": "S01E01 Reasons",
+  "Name": "S01E01",
   "Version": "0.0.0.1",
   "Codes": [
+    {
+        "CodeID": "code-95f5ec39",
+        "CodeType": "Normal",
+        "DisplayText": "discrimination",
+        "NumericValue": 0,
+        "StringValue": "discrimination",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-e5c090d6",
+        "CodeType": "Normal",
+        "DisplayText": "need for cultural change",
+        "NumericValue": 1,
+        "StringValue": "need_for_cultural_change",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-f1d76a3c",
+        "CodeType": "Normal",
+        "DisplayText": "nepotism",
+        "NumericValue": 2,
+        "StringValue": "nepotism",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-0a5bc746",
+        "CodeType": "Normal",
+        "DisplayText": "such issues exist",
+        "NumericValue": 3,
+        "StringValue": "such_issues_exist",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-aff98330",
+        "CodeType": "Normal",
+        "DisplayText": "clannism",
+        "NumericValue": 4,
+        "StringValue": "clannism",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-7a79977b",
+        "CodeType": "Normal",
+        "DisplayText": "gender discrimination",
+        "NumericValue": 5,
+        "StringValue": "gender_discrimination",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-fb1c06d5",
+        "CodeType": "Normal",
+        "DisplayText": "women exclusion from decision making",
```

Comment:
excluded?

Reply:
Will keep it as is it since covers the question context well.